app/controllers/admin/organizations_controller.rb

```diff
@@ -36,10 +36,12 @@ def recover
 
   def new
     @organization = Organization.new
+    render layout: "new_admin"
   end
 
   def edit
     @embedable_email = @organization.auto_user.email if @organization.auto_user
+    render layout: "new_admin"
```

Comment:
The layout also needs to be added to the `create` and `update` actions, so if there is an error it still renders nicely